pkg/test/verify/cluster.go

```diff
@@ -43,19 +43,22 @@ var testCases = map[string]struct {
 }
 
 func AddToolchainClusterAsMember(t *testing.T, functionToVerify FunctionToVerify) {
+	t.Helper()
 	// given
 	defer gock.Off()
 	status := test.NewClusterStatus(toolchainv1alpha1.ConditionReady, corev1.ConditionTrue)
 
 	t.Run("add member ToolchainCluster with namespace label set", func(t *testing.T) {
 		for testName, data := range testCases {
 			t.Run(testName, func(t *testing.T) {
-
-				toolchainCluster, sec := test.NewToolchainCluster("east", test.HostOperatorNs, "secret", status, Labels("member-ns", test.NameHost))
+				toolchainCluster, sec := test.NewToolchainCluster(t, "east", test.HostOperatorNs, "member-ns", "secret", status, data.insecure)
 				// the caBundle should be always ignored
 				toolchainCluster.Spec.CABundle = "ZHVtbXk="
 				toolchainCluster.Spec.DisabledTLSValidations = data.disabledTLSValidations
 
+				if toolchainCluster.Labels == nil {
+					toolchainCluster.Labels = map[string]string{}
+				}
 				toolchainCluster.Labels[cluster.RoleLabel(cluster.Tenant)] = ""
```

Comment:
I guess that this can be removed - we don't rely on cluster role labels anymore, they are in SPC. 
it can be done as a separate PR since there seems to be a few more places where it is still used.